CHANGES.md

```diff
@@ -5,6 +5,7 @@
 ### _Black_
 
 - Fixed Python 3.10 support on platforms without ProcessPoolExecutor (#2631)
+- Fixed `match` statements with open sequence subjects (#2639)
```

Comment:
```suggestion
- Fixed `match` statements with open sequence subjects, like `match a, b:` (#2639)
```
I feel like most users wouldn't understand what an "open sequence subject" is.

Reply:
Makes sense, thanks!